### ISSUE TITLE
HTML compression

### ISSUE THREAD

**Issue by flosse** [cobalt-org/cobalt.rs#98]
It would be great if there is a option to compress the HTML output.
Does s.o. know how to achieve this with a minimal effort?

**Comment by LucioFranco** [comment]
@flosse Are you talking about minimizing the html,css,js etc?

**Comment by flosse** [comment]
> Are you talking about minimizing the html,css,js etc?

exactly!

**Comment by LucioFranco** [comment]
I don't think there is a good way to do it in rust right now. There are a few CSSParser libraries and HTML parser libraries but that would be a lot of work. What I might suggest is using something like gulp on travis when the site is deployed. Gulp can minimize everything very easily.

**Comment by johannhof** [comment]
Unless you're a very high traffic site I would think that compressing the actual HTML will not have a big impact. We can still leave this open.

**Comment by flosse** [comment]
> Unless you're a very high traffic site I would think that compressing the actual HTML will not have a big impact.

You're totally right but besides the actual compression I was thinking about removing comments.
